ISHPullUpSample/ISHPullUpSample/WebViewController.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  WebViewController.swift
+//  ISHPullUpSample
+//
+//  Created by Felix Lamouroux on 30.06.16.
+//  Copyright © 2016 iosphere GmbH. All rights reserved.
+//
+
+import UIKit
+import WebKit
+
+class WebViewController: UIViewController {
+    private weak var webView: WKWebView?
+    private weak var topLabel: UILabel?
+
+    override func loadView() {
+        view = ISHPullUpRoundedView(frame: UIScreen.main().bounds)
```

Comment:
This isn't a hack. We provide a custom backing view for the view controller. Will add a comment why we do that.